linkage-monitor/src/main/java/com/google/cloud/tools/dependencies/linkagemonitor/Analytics.java

```diff
@@ -0,0 +1,223 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.dependencies.linkagemonitor;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.util.concurrent.MoreExecutors;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Report events to Google Analytics.
+ */
+final class Analytics {
+
+  // TODO is this a good value?
+  private static final int DELAY = 773; // a prime number
+
+  private static final Logger logger = Logger.getLogger(Analytics.class.getName());
+
+  private static final String FIRELOG_COLLECTION_URL =
+      "https://firebaselogging-pa.googleapis.com/v1/firelog/legacy/log";
+
+  // Don't change the value; this name is used as an originating "application" of usage metrics.
+  private static final String METRICS_NAME = "cloud-opensource-java-linkage-monitor";
+  
+  // for Concord Clearcut
```

Comment:
"Clearcut" --> "Firelog". We also migrated away from Clearcut to Firelog.

Reply:
I thought we were using the Firelog ingestion service to send data into Clearcut. Are we instead using Firelog without Clearcut?

Reply:
Ah, you may be right. Honestly, I don't really know how the backend works.